src/types/api.js

```diff
@@ -113,5 +117,12 @@
  * @prop {boolean} canLeave
  */
 
+/**
+ * @typedef Analytics
```

Comment:
This `@typedef` was moved to `services/analytics.js` so the one here should be removed.